### PR TITLE
Add fr link for CIRA agreement

### PR DESCRIPTION
**Pull request by deBhal**
CIRA provides the agreement in French, so we might as well offer that to our French speaking users.

This PR is based on #15412, just split out into a separate PR to make sure I don't slow that PR landing.
convention_d_enregistrement_-_titulaire__version_2_0_and_checkout_ _julesauspremiumtest_ _wordpress_com

### CODE REVIEW

File: client/components/domains/registrant-extra-info/ca-form.jsx [hunk 1]
```diff
@@ -179,7 +182,7 @@ class RegistrantExtraInfoCaForm extends React.PureComponent {
 export default connect(
 	state => ( {
 		contactDetailsExtra: getContactDetailsExtraCache( state ),
-		userWpcomLang: getCurrentUserLocale( state )
+		userWpcomLangIsFr: getCurrentUserLocale( state ).match( /^fr-?/i )
```
Comment (yoavf) [inline client/components/domains/registrant-extra-info/ca-form.jsx]: What happens if getCurrentUserLocale returns null? Not introduced here, this is also relevant to current version.

Comment (deBhal) [inline client/components/domains/registrant-extra-info/ca-form.jsx]: If the user is not in the state, the null here will error. I suspect a catastrophic one :)

That's not a problem for checkout (you can't get this deep without having a valid user) but we'll need to look into it more closely if we want to use it for nux.

**Comment by apeatling** [comment]
@yoavf & @deBhal What's the status of this one? Can we merge or close?

**Comment by apeatling** [comment]
Closing this, please reopen if a review is added, or there's more to commit.

**Pull request closed without merging**